qiskit_ibm_provider/ibm_backend_service.py

```diff
@@ -225,6 +226,7 @@ def jobs(
             descending: If ``True``, return the jobs in descending order of the job
                 creation date (i.e. newest first) until the limit is reached.
             instance: The provider in the hub/group/project format.
+            legacy: Retrieve jobs run from the deprecated `qiskit-ibmq-provider`.
```

Comment:
Probably worth noting that _only_ legacy jobs will be returned if `legacy=True` is specified.